records/2022-Q4-activity-report.md

```diff
@@ -0,0 +1,420 @@
+
+### Compiler Academy
+
+For Scala 3.
+
+Scala 3 Compiler Academy is an effort to provide community enthusiasts with a well-defined way to get started contributing to the Scala 3 compiler. We've been incubating it since Summer 2021. The online pair programming sessions at first took place only between the core team members. Throughout 2022, we were welcoming more and more external contributors to participate, but the project was still not open to the public.
+
+In Q4, we're finally opening it to the wider public. We've integrated the project with Scala's official Discord so that everyone present there who is interested in contributing to the compiler will be aware of it and will be able to join.
+
+The significance of the project is to ensure the long-term sustainability of Scala 3. Scala 3's team is smaller than corresponding teams of many other programming languages. We need to ensure that the project is not dependent on a small number of people, but rather on a large number of people who are interested in contributing to the compiler. Compiler Academy is one of the avenues that ensure a stream of new compiler enthusiasts.
+
+You can learn more about the Academy [here](https://compileracademy.carrd.co/). And [here](https://www.scala-lang.org/blog/2022/11/02/compiler-academy.html) is the blog post announcing opening it up.
+
+### Community Expansion
+
+For Scala 2 and 3.
+
+Scala is developed at EPFL, the Polytechnic University of Lausanne, Switzerland. Despite being around for close to 20 years, Scala is still not widely used in the scientific community.
+
+To spark the interest of EPFL scientists and students in Scala, throughout 2022, we developed a workshop that introduces Scala to scientists in a gamified way. The workshop is about programming spacecrafts' trajectories in a game called [Simple Rockets 2](https://www.simplerockets.com/).
+
+In Q4, we finalized this project by conducting the first workshop for the EPFL community. We had around 10 participants, and the workshop was a success. We're tentatively planning to conduct one such workshop per semester to spread awareness about Scala at EPFL and gain more userbase at our home university.
+
+- [EPFL Announcement](https://memento.epfl.ch/event/launching-rockets-with-scala/)
+- [Technical blog](https://akmetiuk.com/posts/2022-02-19-launching-rockets.html) to get an idea of what it is all about.
+
+### Google Summer of Code
+
+For Scala 2 and 3.
+
+Google Summer of Code is another initiative by the Scala Center to give Scala users a way to contribute to OSS. Most of the work on the project was done in Q2-Q3 2022, but there was still some work left to be done. in Q4, all the work was finalized, all of the students graduated successfully, and the project is officially over – for this year. You can read more about how it was and the projects we welcomed in the [blog article](https://www.scala-lang.org/blog/2022/11/15/gsoc-report.html) we published.
```

Comment:
```suggestion
Google Summer of Code is another initiative by the Scala Center to give Scala users a way to contribute to OSS. Most of the work on the project was done in Q2-Q3 2022, but there was still some work left to be done in Q4, all the work was finalized, all of the students graduated successfully, and the project is officially over – for this year. You can read more about how it was and the projects we welcomed in the [blog article](https://www.scala-lang.org/blog/2022/11/15/gsoc-report.html) we published.
```

Reply:
Fixed in fce926e36f94706c5bedd331b59ffca522fecc59